cloud/edgecontroller/Makefile

```diff
@@ -7,3 +7,6 @@ lint:
 	golangci-lint run --disable-all -E gofmt -E golint ./...
 	go vet ./...
 
+generate-code:
```

Comment:
can we name the task to something related to device-crd ? we may have multiple features based on CRD's in future.

Reply:
This task is used for generating code and could be potentially used for not only crds.

Reply:
ok, thanks